content/en/blog/2021/etcd-deployments-on-AWS.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Deployment of etcd on AWS EC2
+author: Caleb Miles, Somoshree Datta
+date: 2021-09-03
```

Comment:
Do we want to aim for a mid month release of this post? Maybe Sept 14th or 15th? @spzala thoughts?

I think we should release the other two in 2-3 week intervals, #469 maybe in early Oct, and #471 in late Oct or early Nov?

Reply:
@nate-double-u since the post is an old one, I am not sure if we want to put the latest date here. The content is great as we already have it but not sure if anything is changed and putting the current date makes it feel like it's the latest blog and tested accordingly.

Reply:
That's a good point. @Okeanos makes a similar one over on the [Platforms/Container-Linux-with-systemd post](https://github.com/etcd-io/website/pull/469#issuecomment-922461700).

I think we should back date this as we are providing this info mostly as an archival reference to a page we're removing.

Which date should we use? The original page creation date? the date it was moved? other?